frontend/src/container/AdvancedSearch.jsx

```diff
@@ -175,6 +187,167 @@ export class AdvancedSearch extends React.Component {
+  }
+
+  handleChangeModifiedRange = (dateObject) => {
+    const { props, state } = this
+    let currentSearch
+
+    if (state.searchType === ADVANCED_SEARCH_TYPE.CONTENT) {
+      currentSearch = props.contentSearch
+    }
+    /*
+      if (state.searchType === ADVANCED_SEARCH_TYPE.USER) {
+        currentSearch= props.userSearch
+      }
+
+      if (state.searchType === ADVANCED_SEARCH_TYPE.SPACE) {
+        currentSearch = props.spaceSearch
+      }
+    */
+
+    let newAppliedModifieddRange = {}
+    const oldAppliedModifieddRange = currentSearch.appliedFilterList.modifiedRange
+
+    if (oldAppliedModifieddRange) {
+      if (Object.keys(dateObject)[0] in oldAppliedModifieddRange) {
+        Object.keys(oldAppliedModifieddRange).forEach(key => {
+          if (key !== Object.keys(dateObject)[0]) {
+            newAppliedModifieddRange[key] = oldAppliedModifieddRange[key]
+          }
+        })
+      } else newAppliedModifieddRange = { ...oldAppliedModifieddRange, ...dateObject }
+    } else newAppliedModifieddRange = dateObject
+
+    props.dispatch(setAppliedFilter(ADVANCED_SEARCH_FILTER.MODIFIED_RANGE, newAppliedModifieddRange, state.searchType))
+
+    this.getSearchResult(
+      { ...currentSearch, searchType: state.searchType },
+      currentSearch.resultList.length,
+      currentSearch.appliedFilterList.searchFieldList,
+      currentSearch.appliedFilterList.createdRange,
+      newAppliedModifieddRange,
+      currentSearch.appliedFilterList.searchFacets
+    )
+  }
+
+  handleChangeSearchFacets = (facetObject) => {
+    const { props, state } = this
+    let currentSearch
+    if (state.searchType === ADVANCED_SEARCH_TYPE.CONTENT) {
+      currentSearch = props.contentSearch
+    }
+    /*
+      if (state.searchType === ADVANCED_SEARCH_TYPE.USER) {
+        currentSearch= props.userSearch
+      }
+
+      if (state.searchType === ADVANCED_SEARCH_TYPE.SPACE) {
+        currentSearch = props.spaceSearch
+      }
+    */
+
+    let newAppliedSearchFacets = {}
+    const oldAppliedSearchFacets = currentSearch.appliedFilterList.searchFacets
+
+    if (oldAppliedSearchFacets) {
+      if (Object.keys(facetObject)[0] in oldAppliedSearchFacets) {
+        Object.keys(oldAppliedSearchFacets).forEach(key => {
+          if (key !== Object.keys(facetObject)[0]) {
+            newAppliedSearchFacets[key] = oldAppliedSearchFacets[key]
+          }
+        })
+      } else newAppliedSearchFacets = { ...oldAppliedSearchFacets, ...facetObject }
+    } else newAppliedSearchFacets = facetObject
+
+    props.dispatch(setAppliedFilter(ADVANCED_SEARCH_FILTER.SEARCH_FACETS, newAppliedSearchFacets, state.searchType))
+
+    this.getSearchResult(
+      { ...currentSearch, searchType: state.searchType },
+      currentSearch.resultList.length,
+      currentSearch.appliedFilterList.searchFieldList,
+      currentSearch.appliedFilterList.createdRange,
+      currentSearch.appliedFilterList.modifiedRange,
+      newAppliedSearchFacets
+    )
```

Comment:
Duplicate of https://github.com/tracim/tracim/pull/4205/files#diff-b55d31b2a6da3b235389870d0f8c87611170ff6eb6e3c4e7b81215b31543e36cR246

Reply:
fix